CHANGELOG_ANDROID.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## [1.9.79](https://github.com/brave/brave-browser/releases/tag/v1.9.79)
+
+- Implemented import/export db options for rewards. ([#9766](https://github.com/brave/brave-browser/issues/))
+- Added settings for `Top sites suggestions` and `Brave suggested sites`. ([#9782](https://github.com/brave/brave-browser/issues/))
+- Added `Estimated Bandwidth Savings` stats on new tab page. ([#8845](https://github.com/brave/brave-browser/issues/))
+- Fixed crash on certain devices when opening custom tabs. ([#9857](https://github.com/brave/brave-browser/issues/))
```

Comment:
Looks like GH issue # at the end of the link is missing which breaks the link/formatting: Example:

Current --> `([#9857](https://github.com/brave/brave-browser/issues/))`
Should be --> `([#9857](https://github.com/brave/brave-browser/issues/9857))`